src/pipeline_v3/cli/management.py

```
@@ -68,110 +70,126 @@
             with contextlib.redirect_stdout(output_capture):
                 if config_path:
                     self.config = PipelineConfig.from_yaml(config_path)
                 else:
                     self.config = PipelineConfig.from_yaml()
-                    
+
             # Check if any error/warning messages were captured
             captured_output = output_capture.getvalue()
             if "Warning:" in captured_output or "Error" in captured_output:
                 # Config loading had issues, create ConfigLoadError
                 logger = logging.getLogger(__name__)
```

Pylint (3.3.7)

[warning] Redefining name 'logger' from outer scope (line 32)

(W0621)